CorsixTH/Lua/persistance.lua

```diff
@@ -255,7 +255,14 @@ end
 --!param filename (string) Path of the file to write.
 function SaveGameFile(filename)
   local data = SaveGame()
-  local f = assert(io.open(filename, "wb"))
+  local f, err = io.open(filename, "wb")
+  if err then
+    local file = os.tmpname()
+    f = io.open(file, "wb")
+    TheApp.ui:addWindow(UIInformation(TheApp.ui,
+        {"Saved " .. filename .. " to " .. file, "Error:", err}))
```

Comment:
This is (potentially) user facing. Should use the language file and make a user friendly ish message.

Reply:
Done, added _S.errors.save_to_tmp.